### PR TITLE
Fix android config args

### PR DESCRIPTION
**Pull request by mribbons**

### CODE REVIEW

File: src/android/webview.kt [hunk 1]
```diff
@@ -52,6 +52,10 @@ open class WebViewClient (activity: WebViewActivity) : android.webkit.WebViewCli
   ): Boolean {
     val url = request.url
 
+    if (url.scheme == "http") {
```
Comment (jwerle) [inline src/android/webview.kt:55]: maybe `https` too?

File: src/android/window.cc [hunk 2]
```diff
@@ -40,6 +40,7 @@ namespace SSC::android {
     options.debug = isDebugEnabled() ? true : false;
     options.env = stream.str();
     options.cwd = rootDirectory.str();
+    options.appData = this->config;
```
Comment (jwerle) [inline src/android/window.cc:43]: Good catch

File: src/cli/cli.cc [hunk 3]
```diff
@@ -1198,6 +1198,16 @@ int main (const int argc, const char* argv[]) {
         }
       }
 
+      if (settings["android_allow_cleartext"].size() == 0)
+      {
```
Comment (jwerle) [inline src/cli/cli.cc:1202]: We prefer left curly braces to be in the same line as the right paren

File: src/cli/cli.cc [hunk 4]
```diff
@@ -1244,6 +1254,7 @@ int main (const int argc, const char* argv[]) {
         split(settings["android_native_sources"], ' ')
       ) {
         auto filename = fs::path(file).filename();
+        log(String("Android NDK source: " + String(targetPath / file)).c_str());
```
Comment (jwerle) [inline src/cli/cli.cc:1257]: Can you remove this? We actually removed it in another commit

**Review by jwerle — CHANGES_REQUESTED** [review]

File: src/android/webview.kt (hunk 1 above)
Comment (mribbons) [reply src/android/webview.kt]: 👍

File: src/android/window.cc (hunk 2 above)
Comment (mribbons) [reply src/android/window.cc:43]: 🙏

File: src/cli/cli.cc (hunk 3 above)
Comment (mribbons) [reply src/cli/cli.cc]: 👍

File: src/cli/cli.cc (hunk 4 above)
Comment (mribbons) [reply src/cli/cli.cc]: 👍

**Review by chicoxyzzy — APPROVED** [review]
This change fixes CI build for Android

**Review by jwerle — APPROVED** [review]

**Pull request merged**